### ISSUE TITLE
Copy paste to initiate image upload

### ISSUE THREAD

**Issue by burtonjc** [waffleio/waffle.io#2900]

**Comment by songawee** [comment]
Could we make this a stretch goal? Seems like it's a nice to have rather than required. If we've got drag and drop, seems like that's a good start. p.s. I have no idea how GitHub gets the filepath from a "paste" event and initiates an upload or encodes the image data to be able to upload.

**Comment by homeyer** [comment]
Yeah I think there's some funky flash dependency for most copy/paste libs

**Comment by harkenn** [comment]
This would be incredibly valuable for us (new users to Waffle) as our devs have to go back to GH in order to paste images there. The drag/drop process is actually slower than going to GH to edit + paste because we are taking screenshots using the OSX/Windows native tools, which don't have drag+drop functionality at all.
